### PR TITLE
[Mobile] Update text on WOFF, mention variable fonts

### PR DESCRIPTION
**Pull request by tidoust**
See #183 and #185.

Variable fonts described as technology in progress in the graphics page with CSS Fonts Loading.

For WOFF, text slightly updated not to mention WOFF 2.0 as "upcoming" as a result and to highlight the benefits for mobile devices.

### CODE REVIEW

File: mobile/graphics.html [hunk 1]
```diff
@@ -43,6 +43,7 @@ <h2>Technologies in progress</h2>
 
         <div data-feature="Downloadable fonts">
           <p>Given the time required for downloading fonts over mobile networks, authors need to adapt their content to the progressive availability of fonts; <a data-featureid="css-font-loading">CSS Font Loading</a> gives the necessary events to developers to enable that adaptation.</p>
+          <p>Font variations along different axes (width, weight, optical size) can be encoded in a single font file. The <a data-featureid="css-fonts-variable">Variable fonts properties</a> in CSS allow finer control over usage of font variants and can further help developers avoid unnecessary network requests to fetch additional variants of a font.</p>
```
Comment (xfq) [inline mobile/graphics.html]: s/Variable fonts properties/font variation properties/ to use the term in the spec.

**Comment by xfq** [comment]
This PR also adds several `.fr.*` files. Was it intentional? I think it might be better to do that in separate commits...

**Comment by tidoust** [comment]
Sorry for the `.fr.*` files which I used locally for testing i18n features and which should never have made it into Git... Now dropped!

**Comment by xfq** [comment]
Thank you!

**Pull request merged**